stream-compositions/cursors/transaction-cursor/src/test/java/com/backbase/stream/compositions/transaction/cursor/core/mapper/TransactionCursorMapperTest.java

```diff
@@ -0,0 +1,101 @@
+package com.backbase.stream.compositions.transaction.cursor.core.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.backbase.stream.compositions.transaction.cursor.core.domain.TransactionCursorEntity;
+import com.backbase.stream.compositions.transaction.cursor.model.TransactionCursor;
+import com.backbase.stream.compositions.transaction.cursor.model.TransactionCursor.StatusEnum;
+import com.backbase.stream.compositions.transaction.cursor.model.TransactionCursorResponse;
+import com.backbase.stream.compositions.transaction.cursor.model.TransactionCursorUpsertRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+
+class TransactionCursorMapperTest {
+
+  TransactionCursorMapper mapper =
+      Mappers.getMapper(TransactionCursorMapper.class);
+
+  @Test
+  void testMapToModel_Success() {
+    TransactionCursorResponse transactionCursorResponse =
+        mapper.mapToModel(getMockDomain());
+    assertNotNull(transactionCursorResponse);
```

Comment:
Good to have some tests here instead of just assertNotNull

Reply:
We can add it future versions.

Reply:
@sunithabackbase @arun-backbase ☝️